src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -175,20 +186,65 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@use "../../../../styles/variables";
-@use "../../../../styles/mixins";
+@use '../../../../styles/variables';
+@use '../../../../styles/mixins';
+@use '../../../../styles/typography';
 
 .qr-code-reader {
-  .icon {
-    color: variables.$color-blue;
+  ::v-deep .header {
+    .qr-scan-wrapper {
+      @include mixins.flex(center, center);
+
+      background-color: variables.$color-bg-1;
+      border: 4px solid rgba(variables.$color-white, 0.05);
+      border-radius: 50%;
+      width: 56px;
+      height: 56px;
+      align-self: center;
+      margin-bottom: 16px;
+
+      .qr-scan {
+        margin-bottom: 0;
+        width: 40px;
+        height: 40px;
+        color: variables.$color-blue;
+      }
+    }
   }
 
-  .camera video {
-    max-width: 70vw;
+  ::v-deep .container .footer .content {
```

Comment:
Same here. And this is a perfect example why using the v-deep is a terrible thing because in my changes I removed the `.footer .content` element.